lib/datasource/cdnjs/index.ts

```diff
@@ -25,33 +25,32 @@ export class CdnJsDatasource extends Datasource {
     // Each library contains multiple assets, so we cache at the library level instead of per-asset
     const library = lookupName.split('/')[0];
     const url = `${registryUrl}libraries/${library}?fields=homepage,repository,assets`;
-    let result: ReleaseResult;
+    let result: ReleaseResult | null = null;
     try {
-      const { assets, homepage, repository } = (
-        await this.http.getJson<CdnjsResponse>(url)
-      ).body;
-      if (!assets) {
-        return null;
-      }
+      const { body } = await this.http.getJson(url);
+      const { assets, homepage, repository } = validResponse(body);
```

Comment:
Does this throw if it's invalid?

Reply:
By the way, we may want to integrate validators with `http.getJson`

Reply:
Sounds interesting, let's create a new issue to discuss details